be/src/storage/task/engine_clone_task.cpp

```diff
@@ -401,7 +401,24 @@ Status EngineCloneTask::_download_files(DataDir* data_dir, const std::string& re
         return Status::OK();
     };
     RETURN_IF_ERROR(HttpClient::execute_with_retry(DOWNLOAD_FILE_MAX_RETRY, 1, list_files_cb));
-    std::vector<string> file_name_list = strings::Split(file_list_str, "\n", strings::SkipWhitespace());
+
+    // Parse file name and size
+    std::vector<string> file_name_list;
+    std::vector<int64_t> file_size_list;
+    for (auto file_str : strings::Split(file_list_str, "\n", strings::SkipWhitespace())) {
```

Comment:
Should keep compatible with old version, otherwise clone task will fail when upgrading. because there are some new version BE and old version BE. when old version send old version message, new version BE should recognize it and handle it correctly.

Reply:
Thanks, that has been fixed.